tests/test_update_manager_no_mocks.py

```
@@ -0,0 +1,444 @@
+            checksum="abc123",
+            metadata={"test": "data"},
+        )
+
+        status = manager.get_database_status()
+        assert isinstance(status, dict)
+        assert "usda" in status
+        assert status["usda"]["version"] == "1.0.0"
+
+    def test_callback_management_real_logic(self, manager):
+        """Test callback management with real callbacks."""
+        callback = lambda x: x  # Real function
```

Flake8 (7.3.0)

[error] do not assign a lambda expression, use a def

(E731)